kubeconfigs/hub

```diff
@@ -0,0 +1,19 @@
+    client-key-data: LS0tLS1CRUdJTiBSU0EgUFJJVkFURSBLRVktLS0tLQpNSUlFcEFJQkFBS0NBUUVBejhrTStaQVJZcnJIbnFtbHVCVk0rRzN5Mzd3bnJkNVpjd1pLQjM1OGNqUTdwU25RCjBOTVVpc3E2T1ZBeUZicXk4elhsMGNnUWFibXB2RHhuSGEzSjBZTk92cy9JSjdJT3BjM2ZMRGNYc0xiRXFvWFkKaFdZek50Um8yODRVdmF1UDNwYnl0dFdtcVozN2FlSG1OQllrZFhwVUNlVHBkS0RoMWRBTW4zL1djUGFZRWcvbQpGa284dzcxclFFV0FxOTNOUmRqSVM5VTE5cE9oNGFGWjF3Vk5VU0pIQjVURTZaaFlqL0VDTVpWWWtRL2FKMjVlCmo2U3BUVVo4VnNUTjQrb3h0cmZmY1A4dkx2TCtiaGo5TVBhYVdjeDVNVmc0SU9rdk01ZVVzQkFKWmJVWXJGVFYKcll5RG51b25uSW5kNXZMTnV0NEZKYlRMYzZjRUZRd0xFdXlQTFFJREFRQUJBb0lCQURnOTJDZzR5YXZxbUp1bQptYmVROUd1Ti94NTJKRnRZZ3VleWZSazRNR0VOS0ZQVlZMTWhYejhPRG9meTRBdW5wRVpXZ1dWZC9LdFJXTk1kCm1sK2JLYlJ2RVVKdXBXT3hPMnUvcExlb0ZkNThLaEZwOXloc2VGSlAvaTM0d3Y1djBKcFVTMUFYMW11MDJsZFgKa09McVYyWUEvWHhNSExDMzEwanpibWRCeC81OUU3YTY2QXJWczhjYk9yUExFdiswZzlsRzhDaDExSHJqbFZUdwpaa1dTYk5oVk93QXg3U05PeVljQTNoUTYyaGRxMnZYdnArcEZhTDNDSUlsdllzZERXUmxYanBROU9HMENUVmxvCm9XdEdrM3VPbU1wZ3JiSjZuNktlSFZhRGVxSUE2U2g2S0RrQnZkQlFqd1UxRnc5REtRK251YmN5eEZYS2RBZ0MKNWp3MkpoVUNnWUVBOHlobVMraTlnTnFzT3FrdVM4V1BEc2ZuMnk1UVBoZEJPVVhHbko1L2p2NDZ4T292YU9DOApXRWRmMkQvQXJaaEhvdFFkdWpmbnBYRkk3Nk1samZNV2pqUFIzeG02YlRFSU9iTGEvTjNlWFkzSWdpUngxR2ZvCmdHOUJCdmxKTjNaNUtaelBHeTBxRmNvbEpJcm9mYlBJSHNrMmpPTE9ubVVzT2FON0RKZlZIUmNDZ1lFQTJzSm0KWWpzWEZHdmpiaXREQ2J6L1RpVlpic2JCN1h4WmsvQmlMTzl1Ymp1Tm1HZkRMSHJBOGNobzZjdWFUV1RxL1pMYwp3bVBlMjFHalF4UWNyMks3aUcyTGVVdEhaMURKV3J5RE1ESDF4MUtobEtSN1A4azRFYTFFQ2VORmd3dUtkQU1pCnQrNHFUcjZ5eng1eHBhT1paYUo4ZFdldGtXdGd6V0dGQXl0aGlGc0NnWUVBNHpGU000Ym8yV1o0R2U5TlZIcm8KcWRUYXJBNWY1MFB2SzBZb3h0bkpNYUFVNVR3eXcvbUh3YXdldjMwQjJMRTU3Y2ZuWnNFcGJZbWxybEp2bW13UgpIUmtKOVR5aTFiZEVpalMxTENOWnkydllkdEQ2a28zcnpXVW1DeEFNK3dkMU9nNUw3SWFNbGdOa0pQcXROQll3CjI2TUJseTJzQkk5ZmZSTmt1MlY2T0UwQ2dZRUEwK2p1L3djWW8ycWpwR3Y2QTZ2MmNML1RjM0RNR1A2VlU2N1AKbTVIUW1xM0x5eS93cTl5NW5UMnpDNDkwaUxaU3dKS01rUzNuaXZMNVFSaHhkU0FpWTlVaC83eUJqYjUvWHdzdgoxZmIvakRjcVdQQkFpQmltM05ESUVQLzhyU1gzMTh2VnpQREVRUVdQOXYwWndBZkxwOTJQekxkRE5wUWE4TGtmCi9XTC9sdk1DZ1lCanZ3VGRxaXpNVTR1aWFHY0pWMHlFVHRObWtMR3hXWUlxSTJNc2E2MktmT2pkL2d4TkZlOTUKWjM1bVhqK3NqTVBrMElub2orZmZDWHZZRW9IMyt5djg2ekVmZVFLRThzY2NnVlI2WFYvR3RLcTFPbUNtckVUegovQldkWU5aV0kzeGFMZDUvSmpRSEQwclFMdWlzdDJYTGwvRFVsdlVQZ0w0N285aWI4WGl4TkE9PQotLS0tLUVORCBSU0EgUFJJVkFURSBLRVktLS0tLQo=
```

Comment:
@ianzhang366 Seems we have to override this file (kubeconfigs/hub) when running with different cluster. Is there any specific reason to change the content of this example like this?  Thanks.

Reply:
nope, I tried to add this to .gitignore but somehow it's not ignoring this one...